parent/pom.xml

```
@@ -64,7 +64,7 @@
     <version.wildfly26.core>18.0.4.Final</version.wildfly26.core>
 
     <version.tomcat9>9.0.90</version.tomcat9>
-    <version.tomcat>10.1.25</version.tomcat>
+    <version.tomcat>10.1.29</version.tomcat>
```

💭 Should we use `10.1.30`? Noticed the respective Github [tag](https://github.com/apache/tomcat/tree/10.1.30) exists, built it locally and was able to fetch it.
```suggestion
    <version.tomcat>10.1.30</version.tomcat>
```

---

Definitely! Nice catch! 👍